Fortnox.NET/WebSockets/Models/WebSocketEvent.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FortnoxNET.WebSockets.Models
+{
+    public class WebSocketEvent
+    {
+        [JsonProperty(PropertyName = "offset")]
+        public string Offset { get; set; }
+
+        [JsonProperty(PropertyName = "tenantId")]
+        public long? TenantId { get; set; }
+
+        [JsonProperty(PropertyName = "topic")]
+        public string Topic { get; set; }
+
+        [JsonProperty(PropertyName = "entityId")]
+        public string EntityId { get; set; }
+
+        // TODO(Oskar): Make this some sort of enum type users can do something with.
```

Comment:
@brokenprogrammer do you want to fix this before merge?

Reply:
I kindof want to wait with this untill I find a better solution to declaring strings as types.